src/tests/test_class.py

```diff
@@ -6,7 +6,7 @@
 import System
 import six
 from Python.Test import ClassTest
-from System.Collections import Hashtable
+from System.Collections import Hashtable

```

Comment:
It was not needed by me, but it seems I had added a windows line-ending. Its fixed now.